examples/cookbook/design/themes/lib/main.dart

```diff
@@ -20,25 +21,25 @@ class MyApp extends StatelessWidget {
 
         // Define the default brightness and colors.
         colorScheme: ColorScheme.fromSwatch(
-          primarySwatch: Colors.purple,
           // TRY THIS: Change to "Brightness.light"
           //           and see that all colors change
           //           to better contrast a light background
+          primarySwatch: Colors.purple,
           brightness: Brightness.dark,
         ),
 
         // Define the default `TextTheme`. Use this to specify the default
         // text styling for headlines, titles, bodies of text, and more.
         textTheme: TextTheme(
+          // TRY THIS: Change one of the GoogleFonts
+          //           to "lato", "poppins", or "lora"
+          //           The title uses "titleLarge"
+          //           and the middle text uses "bodyMedium"
```

Comment:
I think this block should be down a couple lines, above the `titleLarge` and `bodyMedium` lines that it applies to.